datafusion/core/tests/parquet/arrow_statistics.rs

```diff
@@ -164,6 +164,36 @@ impl TestReader {
     }
 }
 
+/// Which statistics should we check?
```

Comment:
I think this is easier to understand (and now it is at least documented why sometimes the page stats are the same and some they are different)